### PR TITLE
add uuid and gnomeVersion

### PR DESCRIPTION
**Pull request by PapyElGringo**

### CODE REVIEW

File: src/manager/msNotificationManager.ts [hunk 1]
```diff
@@ -28,18 +31,24 @@ export class MsNotificationManager extends MsManager {
...
+        }
+
         const previousCheck = Me.stateManager.getState('notification-check')
             ? new Date(Me.stateManager.getState('notification-check'))
             : new Date();
 
         const message = Soup.Message.new(
             'GET',
-            `${API_SERVER}/notifications?lastCheck=${previousCheck.toISOString()}`
+            `${API_SERVER}/notifications?lastCheck=${previousCheck.toISOString()}&uuid=${uuid}&gnomeVersion=${PACKAGE_VERSION}`
```
Comment (HalfVoxel) [inline src/manager/msNotificationManager.ts:46]: Is this the gnome version or the material shell version?

Comment (PapyElGringo) [reply src/manager/msNotificationManager.ts:46]: it's the gnome-shell version

File: src/manager/msNotificationManager.ts [hunk 2]
```diff
@@ -14,11 +14,14 @@ import {
...
 const Dialog = imports.ui.dialog;
 const ModalDialog = imports.ui.modalDialog;
+const { PACKAGE_VERSION } = imports.misc.config;
 
 /** Extension imports */
 const Me = imports.misc.extensionUtils.getCurrentExtension();
 
 const API_SERVER = 'http://api.material-shell.com';
+const beforeGnome43 =
+    compareVersions(gnomeVersionNumber, parseVersion('43.0')) < 0;
```
Comment (HalfVoxel) [inline src/manager/msNotificationManager.ts:24]: Do you even import these?

**Review by HalfVoxel — CHANGES_REQUESTED** [review]

File: src/manager/msNotificationManager.ts (hunk 1 above)
Comment (HalfVoxel) [inline src/manager/msNotificationManager.ts:46]: Is it not reasonable to send material shell version too? While you are at it?

**Review by HalfVoxel — CHANGES_REQUESTED** [review]

File: src/manager/msNotificationManager.ts (hunk 1 above)
Comment (PapyElGringo) [reply src/manager/msNotificationManager.ts:46]: We don't really have a proper Material Shell version  :D

Comment (PapyElGringo) [reply src/manager/msNotificationManager.ts]: ok done !

**Review by HalfVoxel — APPROVED** [review]
Looks good! As long as the server can handle the version being submitted as undefined.

**Pull request merged**